projects/env-semconv.md

```diff
@@ -0,0 +1,69 @@
+### Description
+
+This working group looks to identify and standardize observability for
+sustainability purposes with regards to infrastructure and applications. This
+would look at current tooling around sustainability metrics and standardize the
+meaning and approach for these metrics.
+
+Current efforts of implementations:
+
+- [Kepler](https://github.com/sustainable-computing-io/kepler)
+- [Scaphandre](https://github.com/hubblo-org/scaphandre)
+
+We want to have a unified metrics convention in order to better facilitate measuring sustainability metrics across different ecosystems.
```

Comment:
Just a note/quibble I guess, but I think you should be careful with 'metrics' vs. 'Metrics' in the OpenTelemetry sense. For example, there is no reason that various sustainability metrics can't be attached to other data types in OpenTelemetry (e.g., emitting some sort of co2 usage as a span attribute on a given FaaS function execution).